cmd/vic-machine/create/create.go

```diff
@@ -17,8 +17,10 @@ package create
 import (
```

Comment:
Most of these changes should really live in validator.go rather than in create.go. They will be relevant for later operations such as `update`